VERSION

```diff
@@ -1 +1 @@
-1.18.0
\ No newline at end of file
+1.19.0-dev
```

Comment:
I thought we only update version file during agent release?

Reply:
we only remove the `-dev` bit during release, otherwise we keep this for new agent version dependent functional tests to not break CI